File: tasks/translation/output/Qwen/Qwen2.5-32B-Instruct-AWQ/codenet/direct/comment/comment/direct/Java/Go/s826491904_commented.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
	"strings"
)

func main() {
	var N int
	fmt.Scan(&N)
	scanner := bufio.NewScanner(os.Stdin)
	scanner.Scan()
	inputs := strings.Fields(scanner.Text())
	max, _ := strconv.Atoi(inputs[0])
	min := max

	for i := 1; i < N; i++ {
		scanner.Scan()
		inputs := strings.Fields(scanner.Text())
		num, _ := strconv.Atoi(inputs[0])
		if num > max {
			max = num
		}
		if num < min {
			min = num
		}
	}
	fmt.Println(max - min)
}

// End of Code

File: tasks/translation/output/Qwen/QwQ-32B-AWQ/codenet/mad/Java/C/s826491904_commented.c
#include <stdio.h>

int main() {
    int N, max, min, in, i;
    scanf("%d", &N);
    scanf("%d", &max);
    min = max;
    for (i = 1; i < N; i++) {
        scanf("%d", &in);
        if (in > max) max = in;
        if (in < min) min = in;
    }
    printf("%d\n", max - min);
    return 0;
} // End of Code
